<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View All Tab</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="gotomanagement">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to Management</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
